uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/models/ScoreList.kt

```diff
@@ -124,8 +149,12 @@ class ScoreList {
                     rollingSum += 1.0
                 }
                 if (offset + denominator < values.size) {
-                    if (values[offset + denominator] == Entry.YES_MANUAL) {
-                        rollingSum -= 1.0
+                    val nbOfSkips =
+                        getNumberOfSkipsByInterval(values, offset, offset + denominator)
+                    if (offset + denominator + nbOfSkips < values.size) {
```

Comment:
Create a variable for `offset + denominator + nbOfSkips` to make it easier to read?